packages/b/brpc/xmake.lua

```diff
@@ -5,11 +5,11 @@ package("brpc")
     add_urls("https://github.com/apache/incubator-brpc.git")
     add_versions("1.3.0", "a90cf60714941632b2986826336a7c50cbd3d530")
 
-    add_deps("libzip", "zlib")
     add_deps("leveldb 1.23")
     add_deps("gflags 2.2.2")
     add_deps("protobuf-cpp 3.19.4")
     add_deps("openssl", {system = false})
+    add_deps("libzip", {configs = {system = false}})
```

Comment:
please remove all `system = false`, we should can use system libs.

Reply:
👌